.eslintrc.js

```diff
@@ -41,36 +31,33 @@ module.exports = {
 
         //==============================================================
 
-
         "@typescript-eslint/adjacent-overload-signatures": "error",
         "@typescript-eslint/consistent-type-assertions": "error",
         "@typescript-eslint/member-delimiter-style": "error",
+        "@typescript-eslint/no-empty-function": "error",
```

Comment:
fwiw, I don't think you have to re-add the "recommended" rules below, they are included automatically by `extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"...]`